packages/paste-website/src/pages/components/card/index.mdx

```diff
@@ -0,0 +1,160 @@
+---
+title: Card
+package: '@twilio-paste/card'
+description: The Twilio Paste Card
+---
+
+import {graphql} from 'gatsby';
+import {Box} from '@twilio-paste/box';
+import {Button} from '@twilio-paste/button';
+import {Card, CardFooter} from '@twilio-paste/card';
+import {Heading} from '@twilio-paste/heading';
+import {Text} from '@twilio-paste/text';
+import Changelog from '@twilio-paste/card/CHANGELOG.md';
+import {DoDont, Do, Dont} from '../../../components/DoDont';
+import {Callout, CalloutTitle, CalloutText} from '../../../components/callout';
+import {SidebarCategoryRoutes} from '../../../constants';
+
+export const pageQuery = graphql`
+  {
+    allPasteComponent(filter: {name: {eq: "@twilio-paste/card"}}) {
+      edges {
+        node {
+          name
+          description
+          status
+          version
+        }
+      }
+    }
+    mdx(fields: { slug: { eq: "/card/" } }) {
+      headings {
+        depth
+        value
+      }
+    }
+  }
+`;
+
+<ComponentHeader
+  name="Card"
+  categoryRoute={SidebarCategoryRoutes.COMPONENTS}
+  githubUrl="https://github.com/twilio-labs/paste/blob/master/packages/paste-core/components/card/"
+  storybookUrl="https://twilio-labs.github.io/paste/?path=/story/components-card--default"
+  data={props.data.allPasteComponent.edges}
+/>
+
+---
+
+<contentwrapper>
+
+<TableOfContents headings={props.data.mdx.headings} />
+
+<content>
+
+## Guidelines
+
+### About Cards
+
+Cards are a specifically-styled container that group relatable content and actions. 
+Card is an extremely flexible container that does not require specific components inside of it, 
+allowing an implementer to compose as needed, but elements such as Title, Body, 
+and Button or Anchor are commonly used. See Examples for common Card patterns. 
+A Card does not currently handle any interactive events such as hover, click or focus, 
+though children composed inside of it may commonly have event handlers.
+
+Acknowledging the flexibility of Card component, there are several non-negotiable 
+properties of a Card that differentiate it from a more basic page-layout element, such as Box:
+
+* A background color of $background-color-body.  
```

Comment:
This token is called `$color-background-body`. Might want to double check the others too if this slipped through spec review.